src/Verilator.cpp

```diff
@@ -14,6 +14,8 @@
 //
 //*************************************************************************
 
+#define VERILATOR_VERILATOR_CPP_INTERNAL_
```

Comment:
Maybe we should set this in the makefile instead to make sure no other file likewise eventually includes it?

Reply:
I've also changed define name to ``VERILATOR_INTERNAL_`` as it is no longer defined in ``Verilator.cpp``.